examples/es6/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "homepage": "https://github.com/descope/node-sdk#readme",
   "dependencies": {
-    "@descope/node-sdk": "../..",
+    "@descope/node-sdk": "file:../..",
```

Comment:
just making sure that this will still work for developers want to run this exmaple:
![image](https://user-images.githubusercontent.com/10514677/218300893-73c4ee88-0da2-4d76-b7ee-8ec57cf2b0f0.png)

Reply:
Should be ok